README.md

```diff
@@ -34,13 +36,13 @@ python3 -m venv venv && source venv/bin/activate
 pip install -r requirements.txt && npm install && npm run build
 ```
 
-4. Build all targets at once:
+4. Build your desired platform target.
 
 ```
-make all
+make k8s
```

Comment:
k8s is going to drop a lot of sphinx warnings because of files not referenced in it.
I suggest `make linux` instead.